data_manager.rb

```diff
@@ -60,4 +97,16 @@ def create_label_object(label)
   def create_author_object(author)
     Author.new(author['f_name'], author['l_name'])
   end
+
+  def create_game_object(game)
+    Game.new(game['publish_date'], game['multiplayer'], game['last_played_at'])
+  end
+
+  def create_music_album_object(music_album)
+    MusicAlbum.new(music_album['publish_date'], music_album['on_sportify'])
```

Comment:
I think on_sportify is a typo, shouldn't it be on_spotify?

Reply:
fix on [f9c6a1b]